content/docs/command-reference/data/status.md

```diff
@@ -43,32 +43,37 @@ DVC uncommitted changes:
 (there are other changes not tracked by dvc, use "git status" to see)
 ```
 
-As shown above, the `dvc data status` displays changes in multiple categories:
-
-- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
-  and `.dvc` files but the corresponding cache files are missing.
-- _DVC committed changes_ indicates that there are changes that are
-  `dvc-commit`-ed that differs with the last Git commit. There might be more
-  detailed state on how each of those files changed: _added_, _modified_,
-  _deleted_ and _unknown_.
-- _DVC uncommitted changes_ indicates that there are changes in the working
-  directory that are not `dvc commit`-ed yet. Same as _DVC committed changes_,
-  there might be more detailed state on how each of those files changed.
-- _Untracked files_ shows the files that are not being tracked by DVC and Git.
-  This is disabled by default, unless [`--untracked-files`](#--untracked-files)
-  is specified.
-- _DVC unchanged files_ shows the files that are not changed. This is not shown
-  by default, unless [`--unchanged`](#--unchanged) is specified.
-
-By default, `dvc data status` does not show individual changes inside the
-tracked directories, which can be enabled with [`--granular`](#--granular)
-option.
+`dvc data status` displays changes in multiple categories:
+
+- _Not in cache_ indicates that there are file records (hashes) in `.dvc` or
+  `dvc.lock` files, but the corresponding <abbr>cache</abbr> files are missing.
+  This may happen after cloning a DVC repository but before using `dvc pull` (or
+  `dvc fetch`) to download the data; or after using `dvc gc`.
+
+- _DVC committed changes_ are new, modified, or deleted tracked files or
```

Comment:
Let's call it `added` to keep it in sync with the output.
```suggestion
- _DVC committed changes_ are added, modified, or deleted tracked files or
```

Reply:
Same on _DVC uncommitted changes_.

Reply:
I'm not using back ticks because this doesn't intend to reflect the exact output. I think "added" is misleading and the output itself seems to be up for debate. Please see/comment in https://github.com/iterative/dvc/pull/7943#pullrequestreview-1097028252.

Reply:
@jorgeorpinel It's a good point, but @skshetry also has a reasonable rationale for using "added," so what do you think about continuing discussion in another issue and leaving "added" to get this merged?